src/main/java/icbm/classic/content/blast/BlastBlockModifyHandler.java

```diff
@@ -0,0 +1,45 @@
+package icbm.classic.content.blast;
+
+import icbm.classic.api.events.BlastBlockModifyEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+import java.util.function.Predicate;
+
+@Mod.EventBusSubscriber
+public class BlastBlockModifyHandler {
```

Comment:
This whole class is the wrong way to do events in forge.

Reply:
Out of curiosity, are you referring to the use of the `Predicate` callback? Or the missing `modid = ICBMConstants.DOMAIN`? I have fixed the missing `modid = ICBMConstants.DOMAIN` and can remove the callback part fairly easily.

Reply:
callback and the use of an event to capture event calls. Rather than letting the base code handle a lack of event capture. Event system mostly is built to work in a way that you fire the event, some mod changes the data in the event, and then the base code gets the changes back. Other option is the event is fired, a mod cancels the event, and then the mod does something different.